products/compute/api.yaml

```diff
@@ -1762,29 +1762,15 @@ objects:
           - :AH
           - :SCTP
           - :ICMP
+      # This is a multi-resource resource reference (BackendService (global), RegionBackendService)
+      # We have custom expands that manage this.
       - !ruby/object:Api::Type::ResourceRef
-        # TODO(https://github.com/GoogleCloudPlatform/magic-modules/issues/200)
-        # After adding the RegionBackendService resource, move this field out
-        # of the common fields for forwarding rules. This field should refer
-        # to a RegionBackendService for a ForwardingRule and to a BackendService
-        # for a GlobalForwardingRule.
         name: 'backendService'
         resource: 'BackendService'
         imports: 'selfLink'
         description: |
-          A reference to a BackendService to receive the matched traffic.
-
-          This is used for internal load balancing.
-          (not used for external load balancing)
-      - !ruby/object:Api::Type::Enum
-        name: 'ipVersion'
```

Comment:
Can we use an extra schema entry to deprecate this in Terraform? Even though it's a useless field, it's a breaking change for users that have specified this to get plan-time errors.

Reply:
yep - wasn't sure how to deprecate things until like five minutes ago